### PR TITLE
graphql: add tx rawReceipt field

### PR DESCRIPTION
**Pull request by s1na**
Alternative to https://github.com/ethereum/go-ethereum/pull/24726, also see https://github.com/ethereum/go-ethereum/issues/24720

### CODE REVIEW

File: graphql/schema.go [hunk 1]
```diff
@@ -137,9 +137,11 @@ const schema string = `
         r: BigInt!
         s: BigInt!
         v: BigInt!
-        #Envelope transaction support
+        # Envelope transaction support
         type: Int
         accessList: [AccessTuple!]
+        # RLP-encoding of the receipts
+        receiptsRLP: Bytes!
```
Comment (ryanschneider) [inline graphql/schema.go:144]:
```suggestion
        # binary encoding of the receipts
        receiptsBinary: Bytes!
```

**Review by ryanschneider — COMMENTED** [review]
Some suggestions below based on #24741 where I fixed the issue w/ https://github.com/ethereum/go-ethereum/pull/24726.

**Comment by fjl** [comment]
A better name would be `rawReceipts`.

**Comment by ryanschneider** [comment]
> rawReceipts

sgtm! I’ll change my RPC to debug_getRawReceipt as well.

**Review by holiman — APPROVED** [review]
LGTM

**Pull request merged**